README.md

```diff
@@ -4,12 +4,48 @@ Sunrise Java Shop Framework :sunrise:
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-java.png?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-java) [![Stories in Ready](https://badge.waffle.io/commercetools/commercetools-sunrise-java.png?label=ready&title=Ready)](https://waffle.io/commercetools/commercetools-sunrise-java)
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.commercetools.sunrise/commercetools-sunrise_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.commercetools.sunrise/commercetools-sunrise_2.11)
 
-The next generation shop template.
+The next generation shop framework. 
 
-* Demo: https://demo.commercetools.com
+* [Demo](https://demo.commercetools.com)
 * [Documentation](manual/)
 * [Javadoc](https://commercetools.github.io/commercetools-sunrise-java/javadoc/index.html)
 
+## Installation
+
+### Sunrise Starter Project
+The recommended way to start using Sunrise is to clone the [Sunrise Starter Project](https://github.com/commercetools/commercetools-sunrise-java-starter) and use it as a template project. It already contains all dependencies and configurations needed to run your Sunrise-based project.
```

Comment:
Please make clearer that the dependencies are provided by this framework. otherwise the link between this repo and Sunrise starter is not obvious, I think.